[2024-07-13T02:24:00.376+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-13T02:24:00.396+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_book_api_pipeline.create_table manual__2024-07-13T02:23:58.486156+00:00 [queued]>
[2024-07-13T02:24:00.401+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_book_api_pipeline.create_table manual__2024-07-13T02:23:58.486156+00:00 [queued]>
[2024-07-13T02:24:00.402+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-13T02:24:00.421+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_table> on 2024-07-13 02:23:58.486156+00:00
[2024-07-13T02:24:00.432+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=813) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-13T02:24:00.434+0000] {standard_task_runner.py:63} INFO - Started process 815 to run task
[2024-07-13T02:24:00.439+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***_book_api_pipeline', 'create_table', 'manual__2024-07-13T02:23:58.486156+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/***_book_api_dags.py', '--cfg-path', '/tmp/tmp4pcwei65']
[2024-07-13T02:24:00.440+0000] {standard_task_runner.py:91} INFO - Job 19: Subtask create_table
[2024-07-13T02:24:00.497+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow_book_api_pipeline.create_table manual__2024-07-13T02:23:58.486156+00:00 [running]> on host e6b306b9c204
[2024-07-13T02:24:00.617+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='zainal' AIRFLOW_CTX_DAG_ID='***_book_api_pipeline' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-13T02:23:58.486156+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-13T02:23:58.486156+00:00'
[2024-07-13T02:24:00.620+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-13T02:24:00.631+0000] {base.py:84} INFO - Using connection ID 'snowflake-conn' for task execution.
[2024-07-13T02:24:00.632+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.1, Python Version: 3.12.4, Platform: Linux-6.5.0-1022-azure-x86_64-with-glibc2.36
[2024-07-13T02:24:00.632+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-07-13T02:24:01.884+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-07-13T02:24:01.884+0000] {sql.py:487} INFO - Running statement: CREATE TABLE IF NOT EXISTS book_table (
        title STRING,
        author STRING,
        publish_date DATE
    ), parameters: None
[2024-07-13T02:24:01.960+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-07-13T02:24:01.961+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-13T02:24:01.961+0000] {snowflake.py:410} INFO - Rows affected: 1
[2024-07-13T02:24:01.961+0000] {snowflake.py:411} INFO - Snowflake query id: 01b5a1b0-0000-c766-0001-1bbe0002b192
[2024-07-13T02:24:02.033+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-07-13T02:24:02.034+0000] {connection.py:762} INFO - closed
[2024-07-13T02:24:02.051+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2024-07-13T02:24:02.076+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-13T02:24:02.076+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-13T02:24:02.084+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***_book_api_pipeline, task_id=create_table, run_id=manual__2024-07-13T02:23:58.486156+00:00, execution_date=20240713T022358, start_date=20240713T022400, end_date=20240713T022402
[2024-07-13T02:24:02.135+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-13T02:24:02.156+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-13T02:24:02.163+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
